From bff4e0f2aa6ae10db7bf54e6a0e1eb9b67069d25 Mon Sep 17 00:00:00 2001
From: Mika Joenpera <mika.joenpera@unikie.com>
Date: Wed, 2 Aug 2023 15:07:35 +0300
Subject: [PATCH] Fix Makefile for nrc driver build for SPI

Two lines commented as not needed for in-built kernel module.

Signed-off-by: Mika Joenpera <mika.joenpera@unikie.com>
---
 drivers/staging/nrc/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/nrc/Makefile b/drivers/staging/nrc/Makefile
index 741f9688ab0b..b6bdcfcb0d10 100755
--- a/drivers/staging/nrc/Makefile
+++ b/drivers/staging/nrc/Makefile
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0
-KDIR ?= /lib/modules/$(shell uname -r)/build
-CHECKPATCH ?= $(KDIR)/scripts/checkpatch.pl
+#KDIR ?= /lib/modules/$(shell uname -r)/build
+#CHECKPATCH ?= $(KDIR)/scripts/checkpatch.pl
 
 #include $(KDIR)/.config
 
-- 
2.41.0

